_articles/de/best-practices.md

```diff
@@ -1,283 +1,319 @@
 ---
 lang: de
-title: Best Practices for Maintainers
-description: Making your life easier as an open source maintainer, from documenting processes to leveraging your community.
+title: Gute Maintainer-Praxis
+description: Erleichtern Sie Ihr Leben als Open-Source-Maintainer! Von der Dokumentation von Prozesses bis zum Einsatz Ihrer Community.
 class: best-practices
 toc:
-  what-does-it-mean-to-be-a-maintainer: "What does it mean to be a maintainer?"
-  documenting-your-processes: "Documenting your processes"
-  learning-to-say-no: "Learning to say no"
-  leverage-your-community: "Leverage your community"
-  bring-in-the-robots: "Bring in the robots"
-  its-okay-to-hit-pause: "It’s okay to hit pause"
+  was-bedeutet-es-eine-software-instand-zu-halten: "Was bedeutet es, eine Software instand zu halten?"
+  dokumentieren-sie-ihre-prozesse: "Dokumentieren Sie Ihre Prozesse"
+  lernen-nein-zu-sagen: "Lernen, nein zu sagen"
+  nutzen-sie-ihre-community: "Nutzen Sie Ihre Community"
+  delegieren-sie-an-roboter: "Delegieren Sie an Roboter"
```

Comment:
maybe: `bringen-sie-roboter-ins-spiel`

Reply:
I wanted to make it sound more professional, than playful.